operations.py

```diff
@@ -0,0 +1,294 @@
+""" Operations """
+from .utils import *
+from connectors.core.connector import get_logger, ConnectorError
+
+import requests
+import logging
+import arrow
+import jmespath
+
+logger = get_logger('Cybereason')
+logger.setLevel(logging.DEBUG)# Comment for prod
```

Comment:
should be WARN for release version

Reply:
I just use it to override the default config.ini level when troubleshooting a specific connector, this should be removed